### ISSUE TITLE
Doorbell Motion Switch

### ISSUE THREAD

**Issue by johnluber** [hjdhjd/homebridge-unifi-protect#265]
**Describe The Problem:**

I am trying to remove the doorbell motion switch from HomeKit. I’ve tried adding both Disable.Motion.Switch and Disable.Motion.Switch.MACADDRESS as options but in both cases the motion switch still appears in HomeKit.

Am I misunderstanding this option or using it wrong?

**To Reproduce:**
Add option and restart homebridge.  HomeKit does not change behavior.

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Homebridge Version:** 1.2.5
* **Node Version:** 15.6.0
* **Homebridge-UniFi-Protect Plugin Version:** 3.6.3
* **UniFi Protect Version:** 1.17.3
* **UniFi Protect Controller Type:** UDMP
* **UniFi Protect Controller Firmware Version:** 
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:**
* **Operating System and OS Version:**

**Comment by hjdhjd** [comment]
Hi there - would love to help, but the above is not remotely enough to help. Please post more complete information - as requested in the bug template - in order to get support.

**Comment by johnluber** [comment]
Ok, makes sense

Here’s the snippet of the config I added vi the Homebridge UI:

            "options": [
                "Enable.Motion.SmartDetect",
                "Enable.Doorbell.Trigger.E063DAEFFB49"
            ],

My understanding is that this should add a switch to HomeKit that could be used to simulate a doorbell press.  However, I never see a switch added.  This is the same as if I try to disable the doorbell event switch.

There is nothing logged in either case.  There are no errors and no indication that the change is trying to do anything.

BTW, thank you for all of your efforts on the myq plugin.  I use that one as well and I really appreciate the time you put into into it over the last weeks.

**Comment by hjdhjd** [comment]
Your welcome! Thanks for the feedback.

The plugin logs things prolifically, please provide everything it shows...just because you may not think it’s related, doesn’t mean it isn’t. :smile:

But based just the above...yep, things look fine and should work. The switches are added, but I’m guessing you’re relatively new to HomeKit or haven’t looked at some of the more sophisticated things you can do?

HomeKit groups related services together for a single accessory. Here’s how things work:

HomeKit creates an **accessory**, like say your camera. That camera then hosts a set of **services** on the accessory that provide functionality. Some of these services are viewable by the user, others aren’t. 

What  you see as tiles in the Home app are **accessories**. To view the services they’re connected to, you need to long-press on a tile to see what they contain. You can ask HomeKit to display services as separate tiles as well, on a per-accessory basis. You can read more about that [here along with pictures of how to group and ungroup tiles](https://www.imore.com/how-ungroup-homekit-accessories-home-app).

You added a doorbell trigger to the camera accessory. It’s there, it’s just not going to show up as a different accessory by default, because it’s a service on an existing accessory.

The other alternative is that something is up in your setup, and I need to see the logs to comment further on that.

**Comment by johnluber** [comment]
So, Im not actually new to HomeKit.  I've been using for a number of years and have about 100 or so devices configured.

That said, though I did figure what is going on and it's actually kind of stupid.  I was using homebridge-unifi-protect2 vs home bridge-unifi-protect.  Once I figured that out then everything is now working as expected/documented.

Thanks for your time and for nudging me to look a bit closer at what was configured.
